src/main/resources/templates/db_scripts/schema.json

```diff
@@ -430,6 +437,17 @@
           "targetTableAlias": "purchaseOrder",
           "tableAlias": "titles"
         }
+      ],
+      "index": [
+        {
+          "fieldName": "acqUnitIds",
+          "caseSensitive": false
+        }
+      ],
+      "fullTextIndex": [
+        {
+          "fieldName": "acqUnitIds"
+        }
```

Comment:
The same configuration of indexes for acq units as for Purchase Order, resolves these 2 WARN logs:
`10:29:15 [815929/orders;164360/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing FT search without index for titles.jsonb->>'acqUnitIds', CQL >>> SQL: acqUnitIds = 8535fa1d-3d86-49cf-84ba-df96cd319c2b >>> get_tsvector(f_unaccent(titles.jsonb->>'acqUnitIds')) @@ tsquery_phrase(f_unaccent('8535fa1d-3d86-49cf-84ba-df96cd319c2b'))
` 


`16:48:41 [797007/orders;863055/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing LIKE search without index for titles.jsonb->>'acqUnitIds', CQL >>> SQL: acqUnitIds <> [] >>> lower(f_unaccent(titles.jsonb->>'acqUnitIds')) NOT LIKE lower(f_unaccent('[]'))
`